src/view/options_menu.py

```diff
@@ -1,19 +1,124 @@
-from typing import List, Optional, Any, Dict
+from typing import List, Optional, Dict, Tuple, Any
+from collections import OrderedDict
+
+import pygame
 
 from pygame.event import EventType
 from pygame.surface import Surface
+from pygame.rect import Rect
 
-from view.action import Action
+from view.action import Action, ActionType
 from view.view import View
+from view import colors
 
 
 class OptionsMenu(View):
-    def __init__(
-        self, available: Any, output_dataset: Dict[str, Any], next: Optional[int] = None
-    ) -> None:
+    ARROW_COLOR = colors.ORANGE
+    FONT_COLOR = colors.WHITE
+    LOGO_IMAGE = pygame.image.load("resources/graphics/logo.png")
+    DIVIDER = 0.4
+
+    def __init__(self, tracks: Dict[str, Action], tracks_thumbnails: List[Any]) -> None:
         super().__init__()
+        pygame.font.init()
+        self.font = pygame.font.SysFont("Verdana", 30)
+        self.main_font = pygame.font.SysFont("comicsansms", 60)
```

Comment:
Could these fields be changed to class constants?

Reply:
Just one thing, that `pygame.font.init()` would have to happen at module load time, so probably just before the class declaration (which is not a bad thing).